src/view/configuration/configuration.jsx

```diff
@@ -518,7 +520,22 @@ const Configuration = ({ formikProps, initInfo }) => {
                         />
                       </div>
                     </div>
-
+                    <div className="u-gapTop">
+                      <InfoTipLayout tip="Prehiding snipped should be placed on the page at the very top">
+                        <FieldLabel
+                          labelFor="prehidingStyleField"
+                          label="Default prehiding snippet (optional)"
+                        />
+                      </InfoTipLayout>
+                      <div className="u-gapLeft">
+                        <CopyToClipboard text={prehidingSnippet}>
```

Comment:
This is what I get when I click the button:

```
() => {
  return "!function(e,a,n,t){var i=e.head;if(i){\n      if (a) return;\n      var o=e.createElement(\"style\");\n      o.id=\"alloy-prehiding\",o.innerText=n,i.appendChild(o),\n      setTimeout(function(){o.parentNode&&o.parentNode.removeChild(o)},t)}}\n      (document, document.location.href.indexOf(\"mboxEdit\") !== -1, \".personalization-container { opacity: 0 !important }\", 3000);";
}
```

I don't think that's what you were intending? Should the code they're copying have a `<script>` tag as well?

Reply:
my mistake, should I add `<script></script>` tags?

Reply:
Yes, I think it would be helpful to include a script tag, otherwise it's probable they're going to have to type that out themselves.